MetaMask/metamask-extension
# fix: Migrate users that currently have opensea enabled to have blockaid enabled

## **Description**



[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/23460?quickstart=1)

With Blockaid successfully rolled to users by default and across multiple networks, we will be deprecating the earlier Opensea security alert feature and migrating users to Blockaid.

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/2133

## **Manual testing steps**

1. Go to Settings
2. See that OpenSea + Blockaid is removed

## **Screenshots/Recordings**



### **Before**


https://github.com/MetaMask/metamask-extension/assets/29962968/244189c3-3ef2-432e-be15-7e3098ef4fef



### **After**


https://github.com/MetaMask/metamask-extension/assets/29962968/5be8ddcc-aeb1-4d42-9da4-73ae6b3e94b5



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
